crates/nargo/src/cli/prove_cmd.rs

```diff
@@ -21,29 +24,32 @@ use crate::{
 #[derive(Debug, Clone, Args)]
 pub(crate) struct ProveCommand {
```

Comment:
We should also support passing arguments through the CLI through the `nargo execute` command.